thresher/admin.py

```diff
@@ -1,6 +1,31 @@
 from django.contrib import admin
-from thresher.models import *
+from django.contrib.auth.admin import UserAdmin
+from thresher.models import (Client, Article, AnalysisType,
+							 TUA, Topic, Question, Answer, 
+							 HighlightGroup, MCSubmittedAnswer,
+							 CLSubmittedAnswer, TBSubmittedAnswer, 
+							 DTSubmittedAnswer)
 
+class UserProfileInLine(admin.StackedInline):
+	""" Class meant to serve as an inline in the 
```

Comment:
I still see only one space here?  Or does your editor use TAB characters?  Either way, it should be 4 spaces exactly.